electron/ipc/browser/tabs.ts

```
@@ -165,3 +166,18 @@ ipcMain.handle("tabs:close-tab", async (event, tabId: number) => {
   tab.destroy();
   return true;
 });
+
+ipcMain.handle("tabs:disable-picture-in-picture", async (event) => {
+  if (!browser) return false;
+
+  const sender = event.sender;
+
+  for (const tab of browser?.tabs.tabs.values()) {
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)